README.md

```diff
@@ -235,3 +235,13 @@ Interaction with GNU `make` jobserver
 sccache provides support for a [GNU make jobserver](https://www.gnu.org/software/make/manual/html_node/Job-Slots.html). When the server is started from a process that provides a jobserver, sccache will use that jobserver and provide it to any processes it spawns. (If you are running sccache from a GNU make recipe, you will need to prefix the command with `+` to get this behavior.) If the sccache server is started without a jobserver present it will create its own with the number of slots equal to the number of available CPU cores.
 
 This is most useful when using sccache for Rust compilation, as rustc supports using a jobserver for parallel codegen, so this ensures that rustc will not overwhelm the system with codegen tasks. Cargo implements its own jobserver ([see the information on `NUM_JOBS` in the cargo documentation](https://doc.rust-lang.org/stable/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts)) for rustc to use, so using sccache for Rust compilation in cargo via `RUSTC_WRAPPER` should do the right thing automatically.
+
+---
+
+Known Caveats
+-------------
+
+* `proc-macro` crates like `serde_derive` cannot be cached.
+* `bin` crates cannot be cached.
```

Comment:
Like @luser said, I think this is covered in the Rust-specific documentation, but it's maybe less than obvious that `--emit` controls this; perhaps the Rust-specific documentation should be updated instead?

Reply:
I wrote the Rust caveats while thinking about implementation details. It could stand to be rewritten to be understandable from the perspective of someone using sccache via cargo, perhaps something like "sccache cannot cache compiles of crates that invoke the system linker, such as `bin`, `dylib`, `cdylib`, or `proc-macro` crates".

Reply:
I didn't actually know there was any documentation other than the Readme, because it isn't linked anywhere. I'll add a link.